openslides_backend/models/motion_category.py

```diff
@@ -0,0 +1,16 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionCategory(Model):
+    """
+    Model for motion category.
+    """
+
+    # TODO: add all fields.
+    collection = Collection("motion_category")
+    verbose_name = "motion_category"
+
+    id = fields.IdField(description="The id of this motion category.")
+    name = fields.RequiredCharField(description="The name of this motion category.")
```

Comment:
Just have a look [here](https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/docs/models.txt#L475) to see all fields. The first block are the fields and the second block can be added to docstring because they are the auto generated reverse fields.

Reply:
Add all fields from models.